docs/setup_toolchain.md

```diff
@@ -53,8 +53,8 @@ Next step: [Add keyboard support to Arduino](#add-keyboard-support-to-arduino)
 3. On Ubuntu, you will have to prevent ModemManager from attempting to grab the keyboard virtual serial port when it goes into bootloader mode:
 
     ```sh
-    $ wget https://raw.githubusercontent.com/keyboardio/Kaleidoscope/master/etc/99-kaleidoscope.rules
-    $ sudo cp 99-kaleidoscope.rules /etc/udev/rules.d
+    $ wget https://raw.githubusercontent.com/keyboardio/Kaleidoscope/master/etc/60-kaleidoscope.rules
+    $ sudo cp 60-kaleidoscope.rules /etc/udev/rules.d
     $ sudo /etc/init.d/udev reload
```

Comment:
```suggestion
    $ sudo udevadm control -R
```

might be a more distro-agnostic way to do it.  `systemctl reload systemd-udevd.service` is another option, although I don't know off-hand how consistent the service name is across distros.

Reply:
They're the same across all distros. As far as I know, none of them customise the systemd-supplied unit names.

Reply:
Thanks.  I made and pushed the doc change.